README.md

```diff
@@ -11,3 +11,7 @@ implemented as a set of CRDs.
 ```shell
 $ kubectl create -f crd/cluster-registry.yaml
 ```
+
+### Code of Conduct
```

Comment:
I didn't give you a good example for code of conduct; this is the one to emulate: https://github.com/kubernetes-sigs/federation-v2/commit/f88ed7b88a7f4e6e28e5381473d34162c446fd05